src/clj/fluree/db/flake/flake_db.cljc

```diff
@@ -393,12 +394,24 @@
     (match/match-class db fuel-tracker solution s-mch error-ch))
 
   (-activate-alias [db alias']
-    (when (= alias alias')
-      db))
+    (cond
+      (= alias alias')
+      db
+
+      (iri/f-idx-flatrank-ns alias')
+      db
+
+      :else
+      (throw (ex-info (str "Unknown graph alias: " alias')
+                      {:status 400 :error :db/invalid-query}))))
 
   (-aliases [_]
     [alias])
 
+  where/Searcher
```

Comment:
Does `AsyncDb` need to implement this as well?

Reply:
I don't think so, but can't fully recall why I thought that any longer. I probably should have made a comment. I can look at it again.